docs/reference/method/MongoDBChangeStream-current.txt

```diff
@@ -40,8 +40,6 @@ This example reports events while iterating a change stream.
 
 .. code-block:: php
 
-   <?php
```

Comment:
@alcaeus: You'll need to confirm with docs if this is required for their syntax highlighter.

Reply:
I've compared the current docs (left) and the build attached to this PR (right) side-by-side:
![Docs comparison](https://user-images.githubusercontent.com/383198/212006925-37cd0971-6148-446b-a910-f818ffb7bf4f.png)

At least the current version of the docs no longer does syntax highlighting without PHP tags, so we can't merge this for the time being. Maybe the move to snooty may allow us to remove these tags (@schmalliso would be able to comment on this), but for the time being we'll have to close this PR.

Reply:
Thanks for the feedback 👍